src/systemcmds/tests/test_parameters.cpp

```diff
@@ -21,14 +24,22 @@ class ParameterTest : public UnitTest
 	param_t p3{PARAM_INVALID};
 
 	bool _assert_parameter_int_value(param_t param, int32_t expected);
+	bool _assert_parameter_float_value(param_t param, float expected);
+
 	bool _set_all_int_parameters_to(int32_t value);
 
+	// tests on the test parameters (TEST_RC_X, TEST_RC2_X, TEST_1, TEST_2
```

Comment:
These are all int's, should we add a float as well?

Reply:
added TEST_3 float